README.md.in

```diff
@@ -0,0 +1,92 @@
+[![Build Status](https://jenkins.ovirt.org/job/oVirt_ovirt-ansible-collection_standard-check-pr/badge/icon)](https://jenkins.ovirt.org/job/oVirt_ovirt-ansible-collection_standard-check-pr/)
+[![Build Status](https://img.shields.io/badge/docs-latest-blue.svg)](https://docs.ansible.com/ansible/2.10/collections/ovirt/ovirt/index.html)
+
+oVirt ansible collection
```

Comment:
oVirt Ansible Collection

Reply:
okay, but around AH should we have there also oVirt? I can add there @NAME@ but it will be ovirt or rhv (no large letters).
(or can update the automation to add the proper names)

Reply:
oVirt Ansible Collection is common name for both oVirt and RHV